utils/country_utils.js

```diff
@@ -6,122 +6,67 @@ const countryData = require('./countries');
-			long: countryFound.long,
-			flag: `https://raw.githubusercontent.com/NovelCOVID/API/master/assets/flags/${countryFound.iso2.toLowerCase()}.png`
-		};
-	}
-	return nullReturn;
+	return countryFound ? {
+		_id: countryFound.id,
+		country: countryFound.country,
+		iso2: countryFound.iso2,
+		iso3: countryFound.iso3,
+		lat: countryFound.lat,
+		long: countryFound.long,
+		flag: `https://raw.githubusercontent.com/NovelCOVID/API/master/assets/flags/${countryFound.iso2.toLowerCase()}.png`
+	} : nullReturn;
 };
 
 /**
  * Get all Worldometers data of a certain country
- * @param {Array} countries Array of all countries' Worldometers data
- * @param {string} countryNameParam country name, country code, ISO2, ISO3
+ * @param {Array} data Array of all countries Worldometers data
+ * @param {string} nameParam country name, country code, ISO2, ISO3
  * @param {boolean} strictMatching If true, country name must exactly match the standardized country name
- * @returns {Object}
+ * @param {boolean} continentMode Tells the algorithm to either use 'continent' or 'country' as a selector to use in comparison
+ * @returns {Object} country that was found
  */
-const getCountryWorldometersData = (countries, countryNameParam, strictMatching = false) => {
-	const isText = isNaN(countryNameParam);
-	const countryInfo = isText ? getCountryData(countryNameParam) : null;
-	const standardizedCountryName = stringUtils.wordsStandardize(countryInfo && countryInfo.country ? countryInfo.country : countryNameParam);
-	const foundCountry = countries.find((ctry) => {
-		// either name or ISO
-		if (isText) {
-			// check if provided name matches exactly
-			if (strictMatching) {
-				return stringUtils.wordsStandardize(ctry.country) === standardizedCountryName;
-			} else {
-				stringUtils.wordsStandardize(ctry.country).includes(standardizedCountryName);
-			}
-			return (
-				(ctry.countryInfo.iso3 || 'null').toLowerCase() === countryNameParam.toLowerCase()
-				|| (ctry.countryInfo.iso2 || 'null').toLowerCase() === countryNameParam.toLowerCase()
-				|| ((countryNameParam.length > 3 || isCountryException(countryNameParam.toLowerCase()))
-					&& stringUtils.wordsStandardize(ctry.country).includes(standardizedCountryName))
-			);
-		}
-		// number, must be country ID
-		return ctry.countryInfo._id === Number(countryNameParam);
-	});
-	return foundCountry;
+const getWorldometersData = (data, nameParam, strictMatching, continentMode) => {
+	const selector = continentMode ? 'continent' : 'country';
+	const isText = isNaN(nameParam);
+	const countryInfo = isText ? getCountryData(nameParam) : {};
+	const standardizedName = stringUtils.wordsStandardize(countryInfo.country ? countryInfo.country : nameParam);
+	return data.find((ctry) => !isText ? ctry.countryInfo && ctry.countryInfo._id === Number(nameParam) : strictMatching ? stringUtils.wordsStandardize(ctry[selector]) === standardizedName : ((ctry.countryInfo || {}).iso3 || '').toLowerCase() === nameParam.toLowerCase()
```

Comment:
just split this to more lines and remove `/* eslint-disable max-len */` at the top of the file. It's better not to disable linting when we can